src/Adapter/Driver/Oci8/Oci8.php

```diff
@@ -182,19 +235,22 @@ public function createStatement($sqlOrResource = null)
         }
         return $statement;
     }
-
     /**
      * @param  resource $resource
-     * @param  null     $isBuffered
+     * @param  null     $context
      * @return Result
      */
-    public function createResult($resource, $isBuffered = null)
+    public function createResult($resource, $context = null)
```

Comment:
@Maks3w It's not an issue; the value is passed to `Zend\Db\Adapter\Driver\Oci8\Result`, which currently ignores it. Additionally, the interface defining it, `Zend\Db\Adapter\Driver\DriverInterface`, only defines the first argument, `$resource`. As such, the argument is completely unused at this point.